docs/quickstart/install.md

```diff
@@ -153,6 +153,18 @@ above by running the following before any other CLI commands:
 
 - `tanzu config eula accept` to accept the EULA.
 
+****Customer Experience Improvement Program (CEIP) Prompt**** :
+The Tanzu CLI prompts the user to accept participation in CEIP or not.
+
+Systems and users (via automation scripts, for instance) that expect
+non-interactive use of the CLI can avoid being prompted with either of the
+non-interactive use of the CLI can avoid being prompted with the
+above by running the following before any other CLI commands:
+
+- `tanzu config eula accept` to accept the EULA.
+- `tanzu ceip-participation set <true/false>` to set the CEIP participation status.
```

Comment:
IIUC, I think this command will be removed in the future and is only kept as a safety for the moment.
Therefore I don't think we should document it here.
We can just mention the `TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER` variable.